image-layout.md

```diff
@@ -26,7 +26,7 @@ $ find .
 ./oci-layout
 ./refs
 ./refs/v1.0
-./refs/v1.1
+./refs/another-ref
```

Comment:
Good point to have a generic reference. Perhaps it could be something like `./refs/stable-release`?